### DISCUSSION TITLE
Which transformer-image version to use with Parcel 2?

### DISCUSSION THREAD

**Discussion by RichardJECooke** [parcel-bundler/parcel#6035] [Q&A] [answered]
Hi, here is my `package.json`:

```json
"dependencies": {
    "@babel/plugin-proposal-class-properties": "^7.13.0",
    "@parcel/transformer-image": "^2.0.0-nightly.1981",
    "parcel": "^2.0.0-nightly.548",
    "phaser": "^3.53.1"
}
```

I get told by Parcel when compiling:

```
@parcel/package-manager: Could not find module "@parcel/transformer-image" satisfying 2.0.0-nightly.2252+dc09f125.
>    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Found this conflicting local requirement.
```

Which transformer-image version should I use to fix this please?

**Comment by mischnic** [comment, marked as answer]
As it says, `"@parcel/transformer-image": "^2.0.0-nightly.2252"` should work

**Reply by RichardJECooke** [reply]
Thanks, I changed my package.json file to `"@parcel/transformer-image": "^2.0.0-nightly.2252",` did `npm install`  then `npx parcel src/index.html` 
and now I get the error `@parcel/package-manager: Could not find module "@parcel/transformer-image" satisfying 2.0.0-nightly.2247+7eb2b163.`

**Reply by mischnic** [reply]
You could do this:
1. remove the transformer from package.json & `npm install`
2. Run Parcel, it should autoinstall the transformer

**Reply by RichardJECooke** [reply]
Ok, thank you, now it runs. So weird, at some point Parcel told me to install the transformer-image package, that's the only reason I put it in my package.json to begin with. I wonder why it didn't do it itself earlier.
